packages/sdk-ts/src/core/authz/msgs/MsgExec.ts

```diff
@@ -0,0 +1,91 @@
+import { MsgExec as BaseMsgExec } from '@injectivelabs/chain-api/cosmos/authz/v1beta1/tx_pb'
+import { Message } from 'google-protobuf'
+import { Any } from 'google-protobuf/google/protobuf/any_pb'
+
+import { MsgBase } from '../../MsgBase'
+
+export declare namespace MsgExec {
+  export interface Params {
+    grantee: string
+    msgs: MsgBase<unknown, any, Message, unknown, unknown> | MsgBase<unknown, any, Message, unknown, unknown>[]
```

Comment:
These should be a separate abstraction class, similar to what we have to ExecArgs for cosmwasm contracts, you can check a reference codebase [here](https://github.com/InjectiveLabs/injective-ts/tree/master/packages/sdk-ts/src/core/wasm/exec-args)

Reply:
Sorry, what do you mean? Add for every message class ?

Reply:
Yes, you can start by adding the one that you need and then we can extend them further depending on the usecase

Reply:
I need all messages and the further remember that need to add I think it's no good way  :( 

Maybe ? 
import { Msgs , ExchangeMsgs,} from '../../msgs'

msgs: Msgs | ExchangeMsgs

Reply:
Oh, sorry, yeah I misunderstood what `msgs` mean here - let's use `msgs: Msgs | Msgs[]` here.